lua/barbar/highlight.lua

```diff
@@ -139,19 +138,14 @@ function highlight.setup()
   if config.options.highlight_alternate then
     local alternate_hl = {'TabLine', 'StatusLine'}
 
-    local attributes = hl.attributes(alternate_hl) or {}
+    local attributes = hl.definition(alternate_hl) or {}
```

Comment:
The `definition`/`attribute` naming clarification is to fix LSP diagnostics in 3288b1fafd3b0f31de3993d3d874ad6a7691b9e7